README.md

```diff
@@ -25,16 +25,19 @@ Which collectors are used is controlled by the `--collectors.enabled` flag.
 
 Name     | Description
 ---------|------------
+conntrack | Shows conntrack statistics if available.
```

Comment:
This should be listed as does nothing too, mod_conntrack is usually only loaded on routers and stateful firewalls.